src/main/java/uk/gov/hmcts/ethos/replacement/docmosis/service/CaseManagementForCaseWorkerService.java

```diff
@@ -425,4 +439,26 @@ private void sendUpdateSingleCaseECC(String authToken, CaseDetails currentCaseDe
         }
     }
 
+    /**
+     * Calls reference data API to add HMCTSServiceId to supplementary_data to a case.
+     * @param caseDetails Details on the case
+     * @param accessToken authorisation token for reference data api
+     */
+    public void setHmctsServiceIdSupplementary(CaseDetails caseDetails, String accessToken) throws IOException {
+        Map<String, Map<String, Object>> payloadData = Maps.newHashMap();
+        payloadData.put("$set", singletonMap(HMCTS_SERVICE_ID, hmctsServiceId));
+
+        Map<String, Object> payload = Maps.newHashMap();
+        payload.put("supplementary_data_updates", payloadData);
+
+        try {
+            var response = ccdClient.setSupplementaryData(accessToken, payload, caseDetails.getCaseId());
```

Comment:
Were we not avoiding using var ;)

Reply:
Are we? I personally don't mind var, I prefer this to having it span two lines because its just over 120 chars